src/components/RemoveLogoCTA.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { trans } from '$app/common/helpers';
+import { useTranslation } from 'react-i18next';
+import { Link } from './forms';
+import { proPlan } from '$app/common/guards/guards/pro-plan';
+import { enterprisePlan } from '$app/common/guards/guards/enterprise-plan';
+import { route } from '$app/common/helpers/route';
+import { useCurrentUser } from '$app/common/hooks/useCurrentUser';
+import { useAdmin } from '$app/common/hooks/permissions/useHasPermission';
+
+export function RemoveLogoCTA() {
+  const [t] = useTranslation();
+  const user = useCurrentUser();
+
+  const { isOwner } = useAdmin();
+
+  return (
```

Comment:
You can get way cleaner implementation by early return and escape ? and {} hell.

```tsx
if (!proPlan() && !enterprisePlan() && isOwner) {
...
}

return null;

Reply:
@beganovich Sure thing! I refactored it, it looks cleaner now!